website/content/docs/security/acl/acl-system.mdx

```diff
@@ -2,72 +2,57 @@
+![ACL system component relationships](/img/acl-token-policy-rule-relationship.png)
 
-- **ACL Roles** - Roles allow for the grouping of a set of policies and service
-  identities into a reusable higher-level entity that can be applied to many
-  tokens. (Added in Consul 1.5.0)
+## Tokens
 
-- **ACL Service Identities** - Service identities are a policy template for
-  expressing a link to a policy suitable for use in [Consul
-  Connect](/docs/connect). At authorization time this acts like an
-  additional policy was attached, the contents of which are described further
-  below. These are directly attached to tokens and roles and are not
-  independently configured. (Added in Consul 1.5.0)
+Tokens are the core method of authentication in Consul. The `SecretID` of the token
+(often referred to as the ACL token) is an opaque string that is included in requests to
+identify the person or system making the request. The ACL system will look up the ACL
+token and grant or deny access based on the permissions associated with the token.
+Permissions are granted to a token by linking it to [policies](#policies), [roles](#roles),
+[service identities](#service-identities), and [node identities](#node-identities).
 
-- **ACL Node Identities** - Node identities are a policy template for
-  expressing a link to a policy suitable for use as an [Consul `agent` token
-  ](/docs/agent/options#acl_tokens_agent). At authorization time this acts like an
-  additional policy was attached, the contents of which are described further
-  below. These are directly attached to tokens and roles and are not
-  independently configured. (Added in Consul 1.8.1)
+A Token also has an Accessor ID which is used in [Audit Logging](/docs/enterprise/audit-logging)
+and API responses to identify the token without revealing the secret ID.
 
-- **ACL Auth Methods and Binding Rules** - To learn more about these topics,
-  see the [auth methods documentation page](/docs/acl/auth-methods).
+Tokens can be created directly from the [CLI](/commands/acl/token) or [API](/api-docs/acl/tokens).
+[Auth Methods](/docs/security/acl/auth-methods) can be used to dynamically create tokens
+from a trusted external system.
 
-ACL tokens, policies, roles, auth methods, and binding rules are managed by
-Consul operators via Consul's [ACL API](/api/acl/acl),
-[ACL CLI](/commands/acl), or systems like
-[HashiCorp's Vault](https://www.vaultproject.io/docs/secrets/consul).
+Consul HTTP API requests accept a token from an [HTTP Header](/api-docs/index#authentication),
+and CLI requests accept a token from
+[`-token` command line flag](/commands#authentication) or
+[`CONSUL_HTTP_TOKEN_FILE` environment variable](/commands#consul_http_token_file).
 
-If the ACL system becomes inoperable, you can follow the
-[reset procedure](https://learn.hashicorp.com/tutorials/consul/access-control-troubleshoot#reset-the-acl-system) at any time.
 
-### ACL Policies
+## Policies
```

Comment:
```suggestion
## Policies ((#acl-policies))
```
This should preserve the old anchor name so that existing links still point to this content, while also generating a new #policies anchor that can be used for new content going forward.